frontend/src/components/PsyListing/PsyTable.jsx

```diff
@@ -69,54 +108,10 @@ const PsyTable = ({
             data-test-id="psy-table"
             className="fr-mb-3w"
             caption={title}
-          >
-            <thead>
-              <tr key="headers">
-                <th scope="col">Nom</th>
-                <th scope="col">Adresse</th>
-                {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-                <th scope="col" />
-              </tr>
-            </thead>
-            <tbody>
-              {psychologists
-                .slice((page - 1) * 10, page * 10)
```

Comment:
No need to "slice" anymore? With `data` prop, does `Table` handle pagination internaly? If yes, how is it linked with the `Pagination` component?

Reply:
good catch !